staging/istio/files/crd.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Are we abandoning future upgrades from konvoy < 1.6?

Reply:
Yes, Istio is still a staging chart (and a preview addon) expected to land in a post Konvoy 1.6 world as supported.